LightBurnAdapter.py

```diff
@@ -21,7 +21,7 @@
     help='Open the serial port PORT. Most likely this should be one port of a virtual serial port pair like com0com or tty0tty.')
 
 target_device_args = parser.add_mutually_exclusive_group(required=True)
-target_device_args.add_argument('--ip', nargs=1,
+target_device_args.add_argument('--ip',
```

Comment:
`nargs=1` causes `ARGS.ip` to become a list of one item, instead of just the item as a string

Reply:
From [the documentation](https://docs.python.org/3/library/argparse.html#nargs):

> Note that `nargs=1` produces a list of one item. This is different from the default, in which the item is produced by itself.